src/keys/KeyManager.ts

```diff
@@ -364,11 +424,11 @@ class KeyManager {
     duration: number = 31536000,
     subjectAttrsExtra: Array<{ name: string; value: string }> = [],
     issuerAttrsExtra: Array<{ name: string; value: string }> = [],
-  ): Promise<void> {
+  ): Promise<RecoveryCode> {
```

Comment:
As an interesting point here, I noticed that while rebasing, it seemed to favour the original `): Promise<void> {`  from `master` (as opposed to the change made here) without showing any kind of conflict.

Reply:
It should be `Promise<void>`. Recovery code is acquired by `getRecoveryCode()`.

Reply:
Reverting this return type to `void` (as well as `resetRootKeyPair`) based on discussion with @CMCDragonkai. We have a `getRecoveryCode` function that performs this for us. There's no instance where we require the function to explicitly return recovery code.